shared/actions/fs/platform-specific.desktop.js

```diff
@@ -356,8 +356,8 @@ function* platformSpecificSaga(): Saga.SagaGenerator<any, any> {
     FsGen.openPathInSystemFileManager,
     openPathInSystemFileManager
   )
-  yield* Saga.chainGenerator<ConfigGen.SetupEngineListenersPayload | FsGen.FuseStatusPayload>(
-    [ConfigGen.setupEngineListeners, FsGen.fuseStatus],
+  yield* Saga.chainGenerator<ConfigGen.InstallerRanPayload | FsGen.FuseStatusPayload>(
```

Comment:
this is a close proxy for the behavior before

Reply:
Does this always happen regardless of user settings (fuse driver, autostart, etc.)?

Reply:
this matches the logic before. before config saga would listen for InstallerRan and dispatch setupEngineListeners

Reply:
Got it; sounds good!